src/uct/ib/ud/base/ud_def.h

```diff
@@ -118,16 +118,22 @@ Control packet header
 */
 
 typedef struct uct_ud_neth {
-    uint32_t            packet_type;
-    uct_ud_psn_t        psn;
-    uct_ud_psn_t        ack_psn;
+    union {
+        struct {
+            uint32_t            packet_type;
+            uct_ud_psn_t        psn;
+            uct_ud_psn_t        ack_psn;
+        };
+        int32_t status; /* used in case of failure */
```

Comment:
should not be part of neth, which is wire protocol. should be in uct_ud_send_skb. for example, union with len or reduce flags to uint8